ingestion/src/metadata/ingestion/source/database/snowflake.py

```diff
@@ -127,6 +156,7 @@ def create(cls, config_dict, metadata_config: OpenMetadataConnection):
 
     def next_record(self) -> Iterable[Entity]:
         for inspector in self.get_databases():
+            yield from self.get_all_table_tags()
```

Comment:
In case there are no tags attached to any table or column this might result into error. we should make this as `yield from self.get_all_table_tags() or []`